src/buffer/out/textBuffer.cpp

```diff
@@ -901,6 +901,11 @@ void TextBuffer::SetCurrentLineRendition(const LineRendition lineRendition)
         }
         TriggerRedraw(Viewport::FromDimensions({ 0, rowIndex }, { GetSize().Width(), 1 }));
     }
+    // There is some variation in how this was handled by the different DEC
+    // terminals, but the STD 070 reference (on page D-13) makes it clear that
+    // the delayed wrap (aka the Last Column Flag) was expected to be reset when
+    // line rendition controls were executed.
+    GetCursor().ResetDelayEOLWrap();

```

Comment:
This one sticks out to me as maybe a layering violation - the text buffer is acted-upon by AdaptDispatch for the most part, which keeps the VT behavior encapsulated over there. This one looks like a leak through that boundary. Intentional?

Reply:
Yeah, that's a good point. I've moved it now.